scripts/preprocessing/05a-apply_ica.py

```diff
@@ -76,41 +93,69 @@ def apply_ica(subject, session):
                    .loc[tsv_data['status'] == 'bad', 'component']
                    .to_list())
 
-    # Compare ERP/ERF before and after ICA artifact rejection. The evoked
-    # response is calculated across ALL epochs, just like ICA was run on
-    # all epochs, regardless of their respective experimental condition.
-    #
-    # Note that up until now, we haven't actually rejected any ICs from the
-    # epochs.
-    evoked = epochs.average()
-
-    # Plot source time course
-    fig = ica.plot_sources(evoked, show=config.interactive)
-    report.add_figs_to_section(figs=fig,
-                               captions='All ICs - Source time course')
-
-    # Plot original & corrected data
-    fig = ica.plot_overlay(evoked, show=config.interactive)
-    report.add_figs_to_section(figs=fig,
-                               captions=f'Evoked response (across all epochs) '
-                                        f'before and after cleaning via ICA '
-                                        f'({len(ica.exclude)} ICs removed)')
-    report.save(report_fname, overwrite=True, open_browser=False)
-
-    # Now actually reject the components.
-    msg = f'Rejecting ICs: {ica.exclude}'
-    logger.info(gen_log_message(message=msg, step=5, subject=subject,
-                                session=session))
-    epochs_cleaned = ica.apply(epochs.copy())  # Copy b/c works in-place!
-    epochs_cleaned.apply_baseline(config.baseline)
+    if config.no_epoching:
 
-    msg = 'Saving cleaned epochs.'
-    logger.info(gen_log_message(message=msg, step=5, subject=subject,
-                                session=session))
-    epochs_cleaned.save(fname_epo_out, overwrite=True)
+        msg = f'Rejecting ICs: {ica.exclude}'
+        logger.info(gen_log_message(message=msg, step=5, subject=subject,
```

Comment:
This should be logged for both cases, i.e. should be outside the `if` block

Reply:
yep that's what I intended to do, will fix